dist/build.js

```diff
@@ -189,15 +189,54 @@ Fliplet.Widget.instance('interactive-map', function (widgetData) {
               positionY: marker.data[_this2.markerYPositionColumn]
             }
           };
-        });
+        }); // Check if markers have all the necessary info to be shown
```

Comment:
I don't like that this comment is at the end of a line like this. Is this for the chunk of section above or for what's to come? Or just the one line?

It should be:

1. Above the block of the relevant code
1. At the end of a single line code

Reply:
You are looking at the transpiled file, for some reason comments in new lines get added at the end of the previous line.

Reply:
<img width="729" alt="Screenshot 2019-05-01 at 12 46 01" src="https://user-images.githubusercontent.com/7046481/57015894-1a1e1880-6c0f-11e9-8816-c5cb999d4b28.png">
This is what you see when debugging